django_notify/__init__.py

```diff
@@ -1,40 +0,0 @@
-from django.contrib.contenttypes.models import ContentType
-from django.db.models import Model
```

Comment:
Having this code in the __init__.py file caused AppLoader issues with Django 1.9. I moved the method into the actual model file, below.